cli/internal/deploy/config.go

```diff
@@ -7,6 +7,10 @@ import (
 	"net/url"
 	"path/filepath"
 	"slices"
+	"strings"
+
+	"github.com/BurntSushi/toml"
+	"gopkg.in/yaml.v3"
```

Comment:
This package is no longer maintained. Switch to this one: https://github.com/yaml/go-yaml